### ISSUE TITLE
Initialize love without love window?

### ISSUE THREAD

**Issue by Shadowblitz16** [endlesstravel/Love2dCS#73]
Is there a way to initialize love so that I can use the libraries without having the love window popup?

something like `Love.Special.InitNoWindow()`?

**Comment by endlesstravel** [comment]
no,  it initializes all modules when it starts, so you can't start individual modules directly.

**Comment by Shadowblitz16** [comment]
can something like this be added?

**Comment by endlesstravel** [comment]
This may make  `love.window` `love.graphics` `love.keyboard` `love.mouse` not available. Stopping the activation of a module may cause me to make a lot of changes in coding, and I want to know what it's used for. Can there be other solutions?

**Comment by Shadowblitz16** [comment]
idk if there is other solutions.
I would use it for making engines that can be ran in multiple environments.

for example I could have a game class that runs in the background of another program like winforms.
then I could use drawing, sound and input functions without the main window.

drawing would be somewhat weird since I would have to draw to a love image and then convert it to a system.bitmap and then use it in a picturebox.
